script.js

```diff
@@ -148,4 +153,19 @@ seeProjectButtons.forEach((button, index) => {
       popupContainer.innerHTML = '';
     });
   });
+});
+
+const form = document.querySelector('.contact-form');
+const email = document.querySelector('.email');
+const errorMsg = document.querySelector('.error');
+
+form.addEventListener('submit', (event) => {
+  const emailValue = email.value;
+  const emailchecker = emailValue.replace(/[^a-zA-Z]/g, '').split('');
+  for (let counter = 0; counter < emailchecker.length; counter += 1) {
+    if (emailchecker[counter] === emailchecker[counter].toUpperCase()) {
+      errorMsg.innerText = 'Please enter a correct email address format';
```

Comment:
[Optional] Your error message is a bit vague. It would be much better if you tell the user exactly what is wrong. In this case, a message saying `please enter email in lowercase` is much more descriptive👍